src/analytics.ts

```diff
@@ -255,6 +256,13 @@ export async function viewScreenEvent(
 
 // UI Events
 
+export async function uiErrorEvent(errorInfo: UIErrorInfo): Promise<TrackEvent> {
+    const e = trackEvent('failedTest', 'ui', {
```

Comment:
Intentionally leaving the temporary event here for now while we figure out a more permanent structure for events and metrics